tests/FromCommunity/JsonQueryParameterValidationTest.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+declare(strict_types=1);
+
+namespace League\OpenAPIValidation\Tests\FromCommunity;
```

Comment:
I hope "FromCommunity" namespace exist to pin reproducing cases from different issues discussions into test flow. Since this is a general feature implementation, I suggest you to relocate this code to namespace related to new featuers